test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please make sure that this file is added to the `.gitignore` file and that it's not present in your GitHub repository. Please note that this is part of the project requirements as shown in the screenshot below.

![Screenshot from 2022-11-30 11-22-31](https://user-images.githubusercontent.com/16392046/204744591-18c6b06c-c028-4496-8f09-f89d2c7f6268.png)

You can do this by un-tracking the file in `git` using the following command

```
git rm -r --cached test.md
```
Once the file is un-tracked, add the file to the `.gitignore` file and commit the changes using the `git commit -m` command.